release/models/system/openconfig-system.yang

```diff
@@ -266,6 +272,17 @@ module openconfig-system {
         information such as the current system date and time, uptime,
         last login timestamp, etc.";
     }
+
+    leaf my-mac-address {
```

Comment:
Do we ever expect that there is other configuration of this type? I'm not quite sure that `/system/config` is the best place for this to go - maybe something like `/system/mac-addr/config/local`, you could add other MAC-related configuration here.